include/evmc/evmc.h

```diff
@@ -432,6 +432,52 @@ struct evmc_result
 typedef bool (*evmc_account_exists_fn)(struct evmc_host_context* context,
                                        const evmc_address* address);
 
+/**
+ * Access status per EIP-2929: Gas cost increases for state access opcodes.
+ */
+enum evmc_access_status
+{
+    /**
+     * The entry hasn't been accessed before – it's the first access.
+     */
+    EVMC_COLD_ACCESS = 0,
```

Comment:
Because these identifies are global, it may be better to name these `EVMC_ACCESS_COLD`. Any other opinions?

Reply:
Sure, renamed.